vizro-core/docs/pages/user-guides/filters.md

```diff
@@ -64,7 +64,7 @@ automatically based on the target column, e.g. a dropdown for categorical data,
 
 ## Changing selectors
 
-If you want to have a different selector for your filter, you can provide the `selector` argument of the [`Filter`][vizro.models.Filter] with a different selector model.
+If you want to have a different selector for your filter, you can pass the `selector` argument of the [`Filter`][vizro.models.Filter] with a different selector model.
```

Comment:
```suggestion
If you want to have a different selector for your filter, you can give the `selector` argument of the [`Filter`][vizro.models.Filter] a different selector model.
```

Reply:
@huong-li-nguyen Does this work better? OR 

```suggestion
If you want to have a different selector for your filter, you can pass a different select model as the `selector` argument of the [`Filter`][vizro.models.Filter].
```

Reply:
The first one is better! 👍 so

If you want to have a different selector for your filter, you can give the `selector` argument of the [`Filter`][vizro.models.Filter] a different selector model.